templates/phaser/packages/contracts/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "contracts",
+  "version": "0.0.0",
+  "private": true,
+  "license": "MIT",
+  "scripts": {
+    "build": "forge clean && forge build",
+    "deploy": "pnpm run initialize && mud deploy",
+    "deploy:testnet": "pnpm run initialize && mud deploy --profile=lattice-testnet",
+    "dev": "pnpm mud dev-contracts --tsgenOutput ../client/src/mud",
+    "initialize": "pnpm run tablegen && pnpm run worldgen && pnpm run build && pnpm run worldtypes && pnpm run tsgen",
+    "lint": "pnpm run prettier && pnpm run solhint",
+    "prettier": "prettier --write 'src/**/*.sol'",
+    "solhint": "solhint --config ./.solhint.json 'src/**/*.sol' --fix",
+    "tablegen": "mud tablegen",
+    "test": "mud test",
+    "tsgen": "mud tsgen --configPath mud.config.ts --out ../client/src/mud",
+    "worldgen": "mud worldgen",
+    "worldtypes": "rimraf types && typechain --target=ethers-v5 out/IWorld.sol/IWorld.json"
+  },
+  "devDependencies": {
+    "@ethersproject/abi": "^5.7.0",
+    "@ethersproject/bytes": "^5.7.0",
+    "@ethersproject/providers": "^5.7.2",
+    "@latticexyz/cli": "link:../../../../packages/cli",
+    "@latticexyz/config": "link:../../../../packages/config",
+    "@latticexyz/schema-type": "link:../../../../packages/schema-type",
+    "@latticexyz/solecs": "link:../../../../packages/solecs",
```

Comment:
is the `solecs` this dependency still needed?

Reply:
def not lol

Reply:
also not a peer dependency of ecs browser or sth?